1.6.2 changelog (#19435)
dagster-io/dagster: CHANGES.md

# Changelog

# 1.6.2 (core) / 0.22.2 (libraries)

### New

> - The warning in the Dagster UI to remove unloadable sensors and schedules has now been removed.

Reviewer: (Sorry I wrote this and it look a bit odd to say "remove" twice)
```suggestion
- The warning for unloadable sensors and schedules in the Dagster UI has now been removed.
```
